### ISSUE TITLE
Doesn't start on macOS

### ISSUE THREAD

**Issue by zardoy** [Voxelum/x-minecraft-launcher#135]
![image](https://user-images.githubusercontent.com/46503702/123309798-0a1fcc00-d52e-11eb-82fa-6fbaf401c047.png)

Thats all I see on my macOS 13.

Also it opens devtools on start O_o 
![image](https://user-images.githubusercontent.com/46503702/123310052-58cd6600-d52e-11eb-98fd-d018d7c4c57d.png)

**Comment by zardoy** [comment]
Just wanted to use your launcher because I know that it supports curseforge modpacks. Official curseforge launcher just sucks, and I wanted to use your app, but it didn't even start.

**Comment by ci010** [comment]
😞Sorry I missed this issue. I think the current version doesn't have the issue and could you please try again?

**Comment by ci010** [comment]
Should be fixed in new version.

**Comment by zardoy** [comment]
Sorry for the delay with response, just got time to check. I tried it, and it actually launched, but when I pressed *next* in last step in keystone UI and infinite spinning thing appeared on the button. I remembered that Minecraft wasn't installed, I installed it, tried to relaunch it (cmd+q), macOS complained with crash report and then when I reopened it, no window appears anymore.

<img width="533" alt="Screenshot 2022-04-07 at 11 31 54" src="https://user-images.githubusercontent.com/46503702/162158002-1d3e5e21-1ca6-4b5c-a2ec-ee63d0bc36fa.png">
